packages/pigeon/CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Pigeon Contributor's Guide
+
+**author:** gaaclarke\
+**updated:** 4/6/21
+
+## Description
+
+Pigeon is a code generation tool that adds type safety to Flutter’s Platform
+Channels.  This document serves as an overview of how it functions to help
+people who would like to contribute to the project.
```

Comment:
This seems redundant with the README?

Reply:
Yea, it's one sentence of redundancy that builds up context for the one sentence description.  I think it helps without being too burdensome to those that already have the context.